From: Boyuan Yang <byang@debian.org>
Date: Sat, 4 Jan 2020 22:34:10 -0500
Subject: logo.h: Pre-define SIGRET as "return 0;".

---
 logo.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/logo.h b/logo.h
index 5c29c3f..2ab4516 100644
--- a/logo.h
+++ b/logo.h
@@ -71,7 +71,7 @@
 #endif
 #else
 #define RETSIGTYPE void
-#define SIGRET 
+#define SIGRET return 0;
 #ifndef STDC_HEADERS
 #define STDC_HEADERS
 #endif
