--- a/makefile.in
+++ b/makefile.in
@@ -24,7 +24,7 @@
 logo:	$(OBJS)
 	$(LINKER) $(CFLAGS) $(LDFLAGS) $(OBJS) $(LIBS) -o logo
 
-everything:	logo logolib/Messages helpfiles helpfiles/HELPCONTENTS
+all:	logo logolib/Messages helpfiles helpfiles/HELPCONTENTS
 #logo-mode
 
 mem.o:	mem.c
@@ -77,7 +77,7 @@
 	cp -f helpfiles/* $(LIBLOC)/helpfiles/.
 	cp -f csls/* $(LIBLOC)/csls/.
 #	(cd emacs; prefix=$(prefix) LIBLOC=$(LIBLOC) BINDIR=$(BINDIR) $(MAKE) install)
-	(cd docs; prefix=$(prefix) LIBLOC=$(LIBLOC) BINDIR=$(BINDIR) $(MAKE) install)
+	# (cd docs; prefix=$(prefix) LIBLOC=$(LIBLOC) BINDIR=$(BINDIR) $(MAKE) install)
 #	prefix=$(prefix); LIBLOC=$(LIBLOC); BINDIR=$(BINDIR); export prefix LIBLOC BINDIR; cd emacs; $(MAKE) install
 
 logo-mode: 
